.ci/linux/codestyle.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# This script installs the required tools to be used to make sure there are no code violations in the source code.
+#
+set -euxo pipefail
+
+# Install
+dotnet tool install -g dotnet-format
```

Comment:
```suggestion
dotnet tool install -g dotnet-format --version 3.1.37601
```

Reply:
Just to make sure: with this we pin the version of `dotnet-format` (which is accidentally also `3.x`, but that is unrelated to the .NET Core 3.0 comment from above. I'm ok with this, just want to make sure...